data/pathfinder/dreamscarred_press/path_of_war/pow_abilitycategories_psi.lst

```diff
@@ -3,7 +3,7 @@
 # Original Entry Date: 2015-06-11
 
 # Ability Category						Visible		Editable?		Change Pool?	Fractional values?	Category of Object		Type of Object				Plural description for UI			Display Location
-ABILITYCATEGORY:Pathwalker Discipline			VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:YES	FRACTIONALPOOL:NO		CATEGORY:Internal			TYPE:PathwalkerAbilityChoice		PLURAL:Pathwalker Disciplines			DISPLAYLOCATION:Class Features
+ABILITYCATEGORY:Pathwalker Discipline			VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:YES	FRACTIONALPOOL:NO		CATEGORY:Internal			TYPE:PathwalkerDisciplineChoice		PLURAL:Pathwalker Disciplines			DISPLAYLOCATION:Class Features
```

Comment:
Strangely, the other change is not superficial. The choice of disciplines doesn't appear without it.
Could it be because of `Pathwalker ~ Disciplines` being Internal ability?